src/OrderLinesList/OrderLinesList.js

```diff
@@ -124,6 +125,7 @@ export class OrderLinesList extends Component {
     funds: FUND,
     identifierTypes: IDENTIFIER_TYPES,
     contributorNameTypes: CONTRIBUTOR_NAME_TYPES,
+    invoices: INVOICES,
```

Comment:
Could you please explain the reason of this?

Reply:
Unused resource.  Forgot to remove it.